DependencyInjection/Configuration.php

```diff
@@ -219,7 +219,7 @@
  *   - [bubble]: bool, defaults to true
  *
  * - slackbot:
- *   - team: slack team slug
+ *   - slack_team: slack team slug
```

Comment:
I would made the change in the opposite way to preserve the BC: Just update the Monolog Extension.